shared/constants/fs.js

```diff
@@ -1,16 +1,20 @@
 // @flow
 import * as I from 'immutable'
 import * as Types from './types/fs'
+import {Buffer} from 'buffer'
```

Comment:
In order for this to flowtype correctly we need to assume buffer exists globally. This is true in practice due to our polyfilling. so we should just delete this line and it'll work